ydb/core/persqueue/partition.h

```diff
@@ -664,7 +665,10 @@ class TPartition : public TActorBootstrapped<TPartition> {
     TMaybe<TUsersInfoStorage> UsersInfoStorage;
 
     template <class T> T& GetUserActionAndTransactionEventsFront();
-    template <class T> bool UserActionAndTransactionEventsFrontIs() const;
+    template <class T> T& GetCurrentEvent();
+    TTransaction& GetCurrentTransaction();
+
+    template <class T> void EnsureWhatUserActionAndTransactionEventsFrontIs() const;
```

Comment:
Minor: I'd prefer to call it EnsureUserActionAndTransactionEventsFrontIs, without What. Sounds more natural to me.

Reply:
done
https://github.com/ydb-platform/ydb/pull/3199/commits/b2dfc43f7ecbec298bf3558942512a6733c27e7b